examples/base_cc_vmss_zpa/outputs.tf

```diff
@@ -54,12 +54,22 @@ Bastion Public IP:
 ${module.bastion.public_ip}
 
 
+TB
+
+  testbedconfig_manual_sync_failed = <<TB
+**IMPORTANT (ONLY APPLICABLE FOR INITIATIAL CREATE OF FUNCTION APP)**
```

Comment:
typo INITIATIAL. Might be worth adding an extra "if this is NOT your first run, please ignore" or something like that